FightAndFeast/DataAccess/ProductRepository.cs

```diff
@@ -41,6 +41,33 @@ from Product
         }
 
 
-          }
+        public Product Add(Product newProduct)
+        {
+            
+            using (var db = new SqlConnection(_connectionString))
+            {
+                
+                 
+                var sql = @"Insert INTO [Product]
+                                           ([Name]
+                                            ,[TypeId]
+                                            ,[Price]
+                                            ,[Description])
+                                          output inserted.*
+                                          VALUES
+                                                  (@name
+                                                   ,@typeid
+                                                   ,@price
+                                                   ,@description)";
+
+
+                return db.QueryFirst<Product>(sql, newProduct);
```

Comment:
since we're changing the return type from `Product` to `bool`, this line can be changed to: `return db.Execute(sql, newCustomer) == 1;`